core-bundle/contao/dca/tl_content.php

```diff
@@ -576,14 +576,14 @@
 		'playerStart' => array
 		(
 			'inputType'               => 'text',
-			'eval'                    => array('rgxp'=>'natural', 'nospace'=>true, 'tl_class'=>'w25'),
-			'sql'                     => "int(10) unsigned NOT NULL default 0"
+			'eval'                    => array('maxlength'=>16, 'nospace'=>true, 'tl_class'=>'w25'),
+			'sql'                     => "varchar(16) COLLATE ascii_bin NOT NULL default ''"
```

Comment:
```suggestion
			'sql'                     => "varchar(16) NOT NULL default ''"
```

Why does it have to be `COLLATE ascii_bin`? We only use that for case-sensitive fields.

Reply:
Sorry I didn't know any better, just copied from another varchar field.